src/gui/DatabaseTabWidget.h

```diff
@@ -108,7 +108,7 @@ private slots:
 
     QPointer<DatabaseWidgetStateSync> m_dbWidgetStateSync;
     QPointer<DatabaseWidget> m_dbWidgetPendingLock;
-    QScopedPointer<DatabaseOpenDialog> m_databaseOpenDialog;
+    QPointer<DatabaseOpenDialog> m_databaseOpenDialog;
```

Comment:
Why was this changed?

Reply:
To enable access in testing code. See the other comment.

Reply:
Good to go here too